src/ImageBuilder/test/Start-AzImageBuilderTemplate.Tests.ps1

```diff
@@ -12,7 +12,7 @@ while(-not $mockingPath) {
 . ($mockingPath | Select-Object -First 1).FullName
 
 Describe 'Start-AzImageBuilderTemplate' {
-    It 'Run' {
+    It 'Run' -Skip {
```

Comment:
any reason these are skipped

Reply:
The wait time of the start-az command is about 30 minutes, and a timeout problem occurs during the test case running process, causing the case to fail.